packages/components/src/text-control-with-affixes/style.scss

```diff
@@ -0,0 +1,66 @@
+.text-control-with-affixes {
+	display: inline-flex;
+	flex-direction: row;
+    width: 100%;
```

Comment:
This file is also a mix of tabs & spaces – stylelint is picking it up & flagging it, but the tests are still considered passing. If you run `npm run lint:css` you'll see the issues.

Reply:
Fixed in cc92f19.